src/legacy/core_plugins/metrics/index.ts

```diff
@@ -18,34 +18,36 @@
  */
 
 import { resolve } from 'path';
+import { Legacy } from 'kibana';
+import { PluginInitializerContext } from 'src/core/server';
+import { InternalCoreSetup } from 'src/core/server';
+import { metricsPlugin } from './server/np/';
 
-import { fieldsRoutes } from './server/routes/fields';
-import { visDataRoutes } from './server/routes/vis';
-import { SearchStrategiesRegister } from './server/lib/search_strategies/search_strategies_register';
-
-export default function(kibana) {
+// eslint-disable-next-line import/no-default-export
+export default function(kibana: any) {
   return new kibana.Plugin({
+    id: 'metrics',
```

Comment:
if the plan is to rename to `timeseries`, we should update this & all other references to `metrics`

Reply:
Er, if I'm following the links from #38615 correctly, I guess it was agreed to name TSVB? So I guess `vis_type_tsvb` as originally planned? cc @timroes

Reply:
@timroes ^^

Reply:
i guess we plan to do rename in followup PR ?

Reply:
Oh yes we now renamed it to TSVB instead :D so the function itself can stay like that. Be aware that if you change the id here, that's a breaking user change that can only go into 8.0, since that will change the config option in `kibana.yml` with which you can disable that plugin.